vertical-pod-autoscaler/deploy/admission-controller-deployment.yaml

```diff
@@ -42,6 +42,14 @@ spec:
             - containerPort: 8000
             - name: prometheus
               containerPort: 8944
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /health-check
+              port: prometheus
```

Comment:
not sure if we want to move this to the regular http port but this is where it is currently.